tests/unit/commands/init/test_cli.py

```diff
@@ -2467,3 +2467,46 @@ def does_template_meet_filter_criteria(self):
         }
         dependency_manager = "Gradle"
         self.assertTrue(template_does_not_meet_filter_criteria(app_template, None, dependency_manager, template3))
+
+    @patch("samcli.lib.utils.git_repo.GitRepo")
+    @patch.object(InitTemplates, "__init__", MockInitTemplates.__init__)
+    def test_must_get_local_manifest_path(self, git_repo):
+        template = InitTemplates()
+        template._git_repo.local_path = None
+        manifest_path = template.get_manifest_path()
+        file_name_path = "local_manifest.json"
+        self.assertIn(file_name_path, manifest_path)
+
+    # @patch("samcli.commands.init.init_templates.InitTemplates.get_preprocessed_manifest")
```

Comment:
Are these commented lines okay to remove if unused?

Reply:
Let's remove commented lines if we don't need them anymore